modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/processors/PathsProcessor.java

```diff
@@ -162,6 +162,11 @@ protected void updateLocalRefs(PathItem path, String pathRef) {
     }
 
     protected void updateLocalRefs(ApiResponse response, String pathRef) {
+        if (response.get$ref() != null){
+            if(isLocalRef(response.get$ref())) {
+                response.set$ref(computeLocalRef(response.get$ref(), pathRef));
+            }
+        }
```

Comment:
it makes sense to extract common logic to method like

```
protected void handleLocalRef(String ref, String pathRef) {
  if (ref != null){
    if(isLocalRef(ref)) {
      response.set$ref(computeLocalRef(ref, pathRef));
    }
  }
}
```

replacing the above with:

```
handleLocalRef(response.get$ref(), pathRef);
```